src/shell/Shell.cpp

```diff
@@ -607,7 +638,10 @@ static void printConstVector(wabt::ConstVector& v)
 static void executeInvokeAction(wabt::InvokeAction* action, Walrus::Function* fn, wabt::ConstVector expectedResult,
                                 const char* expectedException, bool expectUserException = false)
 {
-    RELEASE_ASSERT(fn->functionType()->param().size() == action->args.size());
+    if (fn->functionType()->param().size() != action->args.size()) {
+        printf("Error: expected %zu parameter(s) but got %zu.\n", fn->functionType()->param().size(), action->args.size());
+        ASSERT_NOT_REACHED();
```

Comment:
this assert is weaker than RELEASE_ASSERT_NOT_REACHED, keep the latter everywhere